src/overko/state.ts

```diff
@@ -5,9 +5,14 @@ const state = {
     list: [] as Todo[],
     input: ""
   },
-  gender: {
-    checkGender: [],
-    options: [{ name: "Male", value: 1 }, { name: "Female", value: 2 }]
+  platform: {
+    checkPlatform: [],
+    options: [
+      { name: "Option 1", value: 1 },
+      { name: "Option 2", value: 2 },
+      { name: "Option 3", value: 3 },
+      { name: "Option 4", value: 4 }
+    ]
```

Comment:
It would be nice to have an interface for platforms like https://github.com/lrn2prgrm/pyout-frontend/pull/8/commits/8828668bd16c79586faf3ccd6d11a0b50a3a10e6

Reply:
Done! Thank you